spec/spec_helper.rb

```diff
@@ -5,8 +5,14 @@
 require 'fakeweb'
 require 'pry'
 require 'rspec'
+
 require 'simplecov'
-SimpleCov.start
+require 'coveralls' if ENV['CI']
+
+SimpleCov.formatter = Coveralls::SimpleCov::Formatter if ENV['CI']
+SimpleCov.start do
+  add_filter '/spec/'
+end
```

Comment:
The following pattern has worked well in other projects (this example is from Argo):
```
require 'simplecov'
require 'coveralls'
SimpleCov.profiles.define 'dor' do
  add_filter 'gemfiles'
  add_filter 'spec'
end
SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
  SimpleCov::Formatter::HTMLFormatter,
  Coveralls::SimpleCov::Formatter
])
SimpleCov.start 'dor'
```